pincer/middleware/thread_member_update.py

```diff
@@ -27,13 +27,17 @@ async def thread_member_update_middleware(self, payload: GatewayDispatch):
 
     return (
         "on_thread_member_update",
-        ThreadMember.from_dict(construct_client_dict(
-            self,
-            {
-                "join_timestamp": Timestamp(payload.data.pop("join_timestamp")),
-                **payload.data
-            }
-        ))
+        ThreadMember.from_dict(
+            construct_client_dict(
+                self,
+                {
+                    "join_timestamp": Timestamp(
+                        payload.data.pop("join_timestamp")
+                    ),
+                    **payload.data,
+                },
+            )
+        ),
```

Comment:
Would join L34:L36 back together because with all this indentation it's a bit hard to read and it'd be only 80 chars long (I'm guessing this was autopep8)